fmriprep/workflows/bold/confounds.py

```diff
@@ -286,8 +300,24 @@ def _pick_wm(files):
         (rois_plot, outputnode, [('out_report', 'rois_report')]),
     ])
 
-    if use_aroma:
-        # ICA-AROMA
+    if freesurfer:  # Aseg/aparc resampling
+        aseg_tfm = pe.Node(ApplyTransforms(interpolation='NearestNeighbor', float=True),
+                           name='aseg_tfm', mem_gb=0.1)
+        aparc_tfm = pe.Node(ApplyTransforms(interpolation='NearestNeighbor', float=True),
+                            name='aparc_tfm', mem_gb=0.1)
+        workflow.connect([
+            # FreeSurfer ROIs
+            (inputnode, aseg_tfm, [('bold_mask', 'reference_image'),
+                                   ('t1_aseg', 'input_image'),
+                                   ('t1_bold_xform', 'transforms')]),
+            (inputnode, aparc_tfm, [('bold_mask', 'reference_image'),
+                                    ('t1_aparc', 'input_image'),
+                                    ('t1_bold_xform', 'transforms')]),
```

Comment:
This is probably fine, but I wonder if we can do a single-shot transform from `fsnative` to `bold_space-T1w`.

Reply:
Not sure if it is worth spending time on a single transform. Let's go ahead with this and re-iterate if necessary.